EMERITUS.md

```diff
@@ -62,3 +62,12 @@ containerd.
 Leading the development of Windows support in containerd has never been an easy
 task. Kevin has done a great job at both getting the Windows support in a good
 shape and bringing in new Windows contributors to continue to effort.
+
+## Michael Crosby [@crosbymichael](https://github.com/crosbymichael)
+
+Michael played a foundational role in the creation and growth of the containerd project.
+Nearly a decade ago, he architected containerd with a focus on simplicity, robustness,
```

Comment:
"a decade ago" might not be descriptive, as a reader would not know when this sentence was written.